modules/programs/rtorrent.nix

```diff
@@ -0,0 +1,44 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.rtorrent;
+
+in
+
+{
+  meta.maintainers = [ maintainers.marsam ];
+
+  options.programs.rtorrent = {
+    enable = mkEnableOption "rTorrent";
+
+    config = mkOption {
+      type = types.attrsOf types.str;
```

Comment:
Since your example uses integers, perhaps it would make sense to have
```
type = with types; attrsOf (either int str);
```
here?

Reply:
I've changed to `types.lines`; `settings` cannot be attrs, because the rtorrent configuration allows duplicate keys e.g.
```
schedule2 = watch_directory,5,5,load.start=/home/user/watch/*.torrent
schedule2 = untied_directory,5,5,stop_untied=
schedule2 = tied_directory,5,5,start_tied=
```